context/app/static/js/pages/Sample/Sample.tsx

```diff
@@ -30,12 +37,13 @@ function SampleDetail() {
     entity_type,
     descendant_counts,
     ancestor_ids,
+    mapped_data_access_level,
   } = entity;
 
   const [entities, loadingEntities] = useEntitiesData([uuid, ...ancestor_ids]);
```

Comment:
I think `useEntitiesData` accepts a doc as a generic. Could that prevent the assertions below?

Reply:
Good catch, I didn't realize that generic was there 👍🏻